### DISCUSSION TITLE
Latest release is two years old…

### DISCUSSION THREAD

**Discussion by kdambekalns** [thkl/hap-homematic#639] [General]
The latest release is two years old, but there have been a lot of changes to the code base since then. Is there a new release planned? Or do people here just package the source code and invent a new version number?

**Comment by kdambekalns** [comment]
Still no answer here… so, is this project still alive? Does @thkl follow what is happening here? Still using the software? It is totally fine to no longer support something, but it would be great to communicate about that! 🫶

**Reply by blackbasket** [reply]
@thkl is reading if he is mentioned as you did... software is still running fine here... maybe Thomas creates a fresh version, but there were not many changes in this time... only some devices added

**Reply by kdambekalns** [reply]
Well, but those added devices would not be accessible to anyone that can only install a released version. And if there were little changes, rolling a release should be easy, no?

Anyway, glad to hear it's alive (and yes, it does still work.)

**Comment by thkl** [comment]
Hi there, since I’ve started a complete redesign of the configuration Ui (it’s now based on angular) a new release has to wait until this is feature complete.  So i will not break the ability to configure the system.  

Unfortunately I don’t have much time to complete this task at the moment.  

The project is currently not dead but slowly moving…

**Reply by vansemtt** [reply]
okay, waiting for next version

**Reply by phoenixrising87** [reply]
Is there a way of contributing? The plugin is awesome and I'm using it for quite a long time. It would be a shame if it gets outdated or incompatible with homematic/raspberrymatic. 
I would have the necessary angular skills to contribute.

**Reply by mado0803** [reply]
@thkl Is there any chance for external contributors?

**Comment by kuwagei** [comment]
I tried to use it with Raspberrymatic version 3.73.9.20240130 where everything seems to work, but if I want to add some of the bridges to my HomeKit by provided QR Code, it won't be found. So which is the last supported CCU version?

**Reply by jb-home** [reply]
Are the corresponding firewall ports open?
You already have hap bridges connected to HomeKit?
I am currently using it with the latest raspberrymatic version.

Are you using an original CCU3?

**Reply by kuwagei** [reply]
Yes the firewall ports are open, I'm using a Raspberrymatic-Docker-OCI on a Tinkerboard 2, but I also tried it outside a container on a RaspberryPi 3B, which should be very similar to the CCU3. Do you have it working on an original CCU3?

**Reply by jb-home** [reply]
I am just asking, because the original CCU3 has an outdated Node version.
The Rasperrymatic image works fine for me. Try to update the hap-nodejs to the latest version.


/etc/config/rc.d/hap-homematic stop
cd /usr/local/addons/hap-homematic
npm i --cache /tmp/hap-homematic-cache hap-nodejs@latest
/etc/config/rc.d/hap-homematic start
